[OSDOCS#17058]: DC-2: Image Mirroring and oc mirror Tooling-CQA-1 (#115599)
openshift/openshift-docs: modules/installing-mirroring-disconnected-prerequisites.adoc

Reviewer: This isn't bad but let's make this prereqs section like in your other PR using the tip I told you about, to keep it simple:

1. Make this a REFERENCE content type
2. Copy and paste the entire prereqs section in here from the assembly, as-is (keeping all the links)
3. Add an abstract like "Before you mirror images using the oc-mirror plugin, you must meet several prerequisites."
4. In its own paragraph, write "The following prerequisites must be met:"
5. Make sure you delete the additional resources links in the assembly, since those links will now just be in this module.

// Module included in the following assemblies:
//
// * disconnected/installing-mirroring-disconnected.adoc

:_mod-docs-content-type: CONCEPT

> [id="prerequisites_{context}"]

Reviewer 2: ```suggestion
[id="installing-mirroring-disconnected-prerequisites_{context}"]
```
no way that one word will be a valid ID in the repo, always add detail for the content

Reviewer 3: not resolved

> = Prerequisites

Reviewer 2: ```suggestion
= Mirroring in a disconnected environment prerequisites
```
or some such

Reviewer 3: not resolved

[role="_abstract"]

> You must have a container image registry that supports Docker v2-2 in the location that will host the {product-title} cluster, such as {quay}.

Reviewer 2: ```suggestion
You must have a container image registry that supports Docker v2-2 in the location that hosta the {product-title} cluster, such as {quay}.
```
present tense

Reviewer 3: not resolved
